### ISSUE TITLE
Role and Label missing in Navigation screen

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#842]
**Describe the bug**
Role and Label missing for various divs which are acting like buttons at Navigation screen.

**To Reproduce**
1. Go to Navigation screen and select a page for a site.
2. You will see various buttons like "Delete Site", "Copy Site" etc.
3. When we inspect any of these buttons, we will notice that these are divs and not buttons and role is missing for these div's.

**Expected behavior**
Role and Label should be present for all the divs or anchor tags which acts like buttons.

**Additional context**
Navigate all the options available for similar issue and fix.
